lightly_studio/.env.example

```
@@ -15,24 +15,24 @@ LIGHTLY_STUDIO_LICENSE_KEY="your_license_key_here"
 # LIGHTLY_STUDIO_MODEL_CACHE_DIR="/Users/username/.cache/lightly-studio"
 
 # Example script dataset paths
-EXAMPLES_DATASET_PATH="dataset_examples/coco_subset_128_images/images"
-EXAMPLES_YOLO_YAML_PATH="dataset_examples/road_signs_yolo/data.yaml"
+EXAMPLES_DATASET_PATH="datasets/coco_subset_128_images/images"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)